src/test/java/org/mockitousage/constructor/CreatingMocksWithConstructorTest.java

```diff
@@ -52,6 +55,12 @@ public void can_spy_abstract_classes() {
     }
 
     @Test
+    public void can_spy_abstract_classes_with_constructor_args() {
```

Comment:
Options for ambiguity:

1. use 1st constructor that matches
2. throw an exception

I'd say we go for option 1 because it's simpler to implement and seems more convenient (works out of the box for certain use cases).
If we go for option 2 we need to inform the user what to do to resolve the problem (for example document and suggest @fluentfuture idea of creating an inner implementation).